apps/emqx_connector_aggregator/src/emqx_connector_aggregator_schema.erl

```diff
@@ -0,0 +1,89 @@
+%%--------------------------------------------------------------------
+%% Copyright (c) 2024 EMQ Technologies Co., Ltd. All Rights Reserved.
+%%--------------------------------------------------------------------
+
+-module(emqx_connector_aggregator_schema).
+
+-behaviour(hocon_schema).
+
+-include_lib("typerefl/include/types.hrl").
+-include_lib("hocon/include/hoconsc.hrl").
+
+%% API
+-export([
+    container/0
+]).
+
+%% `hocon_schema' API
+-export([
+    namespace/0,
+    roots/0,
+    fields/1,
+    desc/1
+]).
+
+%%------------------------------------------------------------------------------
+%% Type declarations
+%%------------------------------------------------------------------------------
+
+%%------------------------------------------------------------------------------
+%% API
+%%------------------------------------------------------------------------------
+
+container() ->
+    {container,
+        hoconsc:mk(
+            %% TODO: Support selectors once there are more than one container.
```

Comment:
remove?

Reply:
This comment was just transported from its previous location.

Reply:
i.e. also the product of a refactoring